src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -58,7 +58,7 @@ public function getFunctions(): array
             new TwigFunction('form', null, ['node_class' => RenderBlockNode::class, 'is_safe' => ['html']]),
             new TwigFunction('form_start', null, ['node_class' => RenderBlockNode::class, 'is_safe' => ['html']]),
             new TwigFunction('form_end', null, ['node_class' => RenderBlockNode::class, 'is_safe' => ['html']]),
-            new TwigFunction('csrf_token', [FormRenderer::class, 'renderCsrfToken']),
+            new TwigFunction('form_csrf_token', [FormRenderer::class, 'renderCsrfToken']),
```

Comment:
this twig function is also defined by the CrfExtension
that makes it impossible to have a different CSRF token manager for forms vs not-forms use cases (APIs ?)
I guess this change alone would break apps where csrf_protection is disabled while form.csrf_protection is enabled, isn't it? so we'd need to register "csrf_token" and be sure that the one from CsrfExtension wins when enabled.

Reply:
I think Bridge extensions are registered first so..

Reply:
Issue resolved by keeping the csrf_token helper and adding a new form_csrf_token one